server/controllers/submit-controller.js

```diff
@@ -34,13 +34,24 @@ module.exports = ({ strapi }) => ({
     // Adds to DB
     let parsedScore = verification.score || -1
     try {
-      await strapi.query('plugin::ezforms.submission').create({
-        data: {
-          score: parsedScore,
-          data: ctx.request.body.formData,
+      if(strapi.config.get('plugin.ezforms.enableFormName') === true) {
+        await strapi.query('plugin::ezforms.submission').create({
+          data: {
+            score: parsedScore,
+            formName: ctx.request.body.formName,
+            data: ctx.request.body.formData,
+          }
         }
+        )
+      } else {
+        await strapi.query('plugin::ezforms.submission').create({
+          data: {
+            score: parsedScore,
+            data: ctx.request.body.formData,
```

Comment:
There shouldn't be duplicate calls here, I would use a ternary operator to default form name to "Form" if its not provided.